crafts/crafts.html

```diff
@@ -0,0 +1,569 @@
+                </div>
+              </div>
+            </div>
+            
+          </div>
+
+        
+      </div>
+    </nav>
+    
+    <div class="bg-light" style="height: 100vh;">
+      <div class="container py-5">
+        <div class="row h-100 align-items-center py-5">
+          <div class="col-lg-6">
+            <h1 class="fs-1 fw-bold" style="color: rgb(0, 255, 149);">Great <span style="color: rgb(46, 131, 243);">companies</span> are built on great products</h1>
+  
+            <p class="lead text-muted mb-0">
+              Lorem ipsum dolor sit amet, consectetur adipisicing
+              elit, sed do eiusmod tempor incididunt ut labore et
+              dolore magna aliqua.
+              Here are some crafted products made with ❤️ by Iedc crafts.
+            </p><br>
+              <!-- Animated button -->
+              <ul class="list-inline">
+                <a class="animated-btn text-white list-inline-item"><i class="fa fa-play"></i></a>
+                <div class="list-inline-item">
+                    <span class="text-muted d-block line-height-sm">Watch</span><strong class="text-uppercase text-dark">Intro Video</strong>
+                </div>
+              </ul>
+          </div>
+          <div class="col-lg-6 d-none d-lg-block" style="margin-right: 0;padding-left: 112px;"><img src="assets/img/iedc-crafts.png" alt="" class="img-fluid"></div>
+        </div>
+      </div>
+    </div>
+
+  
+      <div class="container py-5">
+
+        <div class="row mb-5">
+          <div class="col-lg-8 mx-auto">
+            <div class="bg-white p-5 rounded shadow">
+            <!-- First Row [Statistics]-->
+            <h2 class="font-weight-bold mb-2">Latest Statistics</h2>
+            <p class="font-italic text-muted mb-4">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.</p>
+            
+        
+        
+            <div class="row row-cols-1 row-cols-md-3 g-4">
+        
+              <div class="col">
+                <div class="card h-100 rounded shadow-sm border-0">
+        
+                    <div class="card-body p-5"><i class="fa fa-tasks fa-2x mb-3 text-success"></i>
+                        <h5>Products Sales</h5>
+        
+                        <div class="progress rounded-pill">
+                            <div role="progressbar" aria-valuenow="80" aria-valuemin="0" aria-valuemax="100" style="width: 80%;" class="progress-bar bg-success rounded-pill"></div>
+                        </div>
+                    </div>
+        
+                </div>
+              </div>
+        
+        
+              <div class="col">
+                <div class="card h-100 rounded shadow-sm border-0">
+        
+                  <div class="card-body p-5"><i class="fa fa-user-circle-o fa-2x mb-3 text-info"></i>
```

Comment:
![image](https://user-images.githubusercontent.com/38415384/123513253-fca73680-d640-11eb-8379-9f7632ec321b.png)

This fA icon is for pro I guess..

Reply:
yeah I just kept that hangin i will update the icon on next push